test/b2gedmntuples_cfg.py

```diff
@@ -654,14 +650,15 @@
 process.skimmedPatMET = cms.EDFilter(
     "PATMETSelector",
     #    src = cms.InputTag(metLabel, "", "PAT"),
-    src = cms.InputTag(metLabel, "", metProcess),
+#    src = cms.InputTag(metLabel, "", metProcess),
+    src = cms.InputTag(metLabel),
     cut = cms.string("")
     )
 
 process.skimmedPatPuppiMET = cms.EDFilter(
     "PATMETSelector",
     #    src = cms.InputTag(metLabel, "", "PAT"),
-    src = cms.InputTag(puppimetLabel, "", metProcess),
+    src = cms.InputTag(puppimetLabel),
```

Comment:
We need the process to be the current one to save the T1 met which is recalculated on-the-fly.

At what level you want to drop the variable?